pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: dart_style
 # Note: See tool/grind.dart for how to bump the version.
-version: 1.3.8+1
```

Comment:
FYI for future changes this should have been `1.3.9` before. We avoid the `+1` once the major version is non-zero to avoid confusion.

Reply:
Oops, right.